src/containers/chat/list-of-users-with-search/ListOfUsersWithSearch.constants.ts

```diff
@@ -0,0 +1,23 @@
+import { ChatResponse } from '~/types'
+
+export const isCorrectUser = (item: ChatResponse, userId: string): boolean => {
+  const correctUser = item.members.filter(
+    (member) => member.user._id !== userId
+  )
+  return correctUser.length > 0
+}
+
+export const filterChats = (
+  listOfChats: ChatResponse[],
+  userId: string,
+  search: string
+): ChatResponse[] => {
+  const lowerCaseSearch = search.toLocaleLowerCase()
+  return listOfChats.filter((item) => {
+    if (!isCorrectUser(item, userId) || !item.latestMessage) return false
+
```

Comment:
Maybe returning an empty array would be a better solution. Because you map the variable which get the result from this function?